akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -104,6 +104,9 @@ class PhiAccrualFailureDetector(
 
   private val acceptableHeartbeatPauseMillis = acceptableHeartbeatPause.toMillis
 
+  // FIXME: fix this binary compatibility hack in 2.6
```

Comment:
Change it to a comment that it was introduced as a var because of binary compatibility constraints.